src/Reports/SubsiteReportWrapper.php

```diff
@@ -2,32 +2,31 @@
 
 namespace SilverStripe\Subsites\Reports;
 
+use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\FieldList;
-use SilverStripe\Forms\TreeMultiselectField;
 use SilverStripe\Reports\ReportWrapper;
 use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Subsites\State\SubsiteState;
 
 /**
  * Creates a subsite-aware version of another report.
  * Pass another report (or its classname) into the constructor.
  */
 class SubsiteReportWrapper extends ReportWrapper
 {
-
     /**
      * @return FieldList
      */
     public function parameterFields()
     {
         $subsites = Subsite::accessible_sites('CMS_ACCESS_CMSMain', true);
-        $options = $subsites->toDropdownMap('ID', 'Title');
+        $options = $subsites->map()->toArray();
 
-        $subsiteField = TreeMultiselectField::create(
-            'Subsites',
-            _t(__CLASS__ . '.ReportDropdown', 'Sites'),
+        $subsiteField = DropdownField::create(
```

Comment:
Using DropdownField instead of TreeMultiselectField as we no longer support passing multiple IDs - see https://github.com/silverstripe/silverstripe-subsites/pull/504